ideascube/serveradmin/package.py

```diff
@@ -0,0 +1,66 @@
+
+
+import yaml
+import os.path as op
+import zipfile
+import hashlib
+import codecs
+import csv
+
+
+class MediaCenterPackage:
+    def __init__(self, working_dir, medias=None):
+        self.working_dir = working_dir
+        self.medias = medias or []
+        self.medias = list(filter(self.check_media, medias or []))
```

Comment:
Why twice self.media = … ?
Only one is useful, you never seen the first line. Look the red light, you never seen this line. :)